user_modules/shared_testcases/testCasesForRAI.lua

```diff
@@ -0,0 +1,169 @@
+local testCasesForRAI = {}
+
+require('atf.util')
+local events = require("events")
+
+--[[@InitHMI_onReady_without_UI_GetCapabilities: replace original InitHMIOnReady from connecttest 
+--! without expect UI.GetCapabilites
+--! @parameters: NO
+--]]
+function testCasesForRAI:InitHMI_onReady_without_UI_GetCapabilities(self)
```

Comment:
@aderiabin, I found the problem.
Done in 732c94b40c3641aea5c276e0227ba4d2d48869ce